src/main/java/frc/robot/commands/OuttakeCommand.java

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.OuttakeSubsystem;
+import frc.robot.subsystems.OuttakeSubsystem.Modes;
+
+public class OuttakeCommand extends CommandBase {
```

Comment:
this command should be renamed to SetOuttakeSubsystemModeCommand for clarity--its long, but it will disambiguate with the intake/system state better.